server/.env.development

```diff
@@ -4,8 +4,12 @@
 GOOGLE_CLOUD_PROJECT=gpay-group-buy
 SERVER_URL=http://localhost:5000
 WEB_URL=http://localhost:3000
+SPOT_MERCHANT_ID=BCR2DN6T3P5NDAKX
+SPOT_MICROAPP_ID=14116337401048649233
 
 # Secret keys
 # Redefine these in .env.development.local with the appropriate values.
 # Secret environment variables for production should be written in app.secret.yaml.
 OAUTH_CLIENT_ID=<Secret>
+SERVICE_ACCOUNT_CLIENT_EMAIL=<Secret>
```

Comment:
do you need to add both service account keys to server/app.yaml?

Reply:
@ifhuang Yep! We need both there. But since this is a secret, I added it to app.secret.yaml which is not tracked by git.